web-console/e2e-tests/cancel-query.spec.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as playwright from 'playwright-chromium';
+
+import { QueryOverview } from './component/query/overview';
+import { saveScreenshotIfError } from './util/debug';
+import { UNIFIED_CONSOLE_URL } from './util/druid';
+import { createBrowser, createPage } from './util/playwright';
+import { waitTillWebConsoleReady } from './util/setup';
+
+jest.setTimeout(5 * 60 * 1000);
+
+describe('Cancel query', () => {
+  let browser: playwright.Browser;
+  let page: playwright.Page;
+
+  beforeAll(async () => {
+    await waitTillWebConsoleReady();
+    browser = await createBrowser();
+  });
+
+  beforeEach(async () => {
+    page = await createPage(browser);
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  it('delete accepted', async () => {
+    const testName = 'cancel-query';
+    await saveScreenshotIfError(testName, page, async () => {
+      await validateCancelQuery(page);
+    });
+  });
+});
+
+async function validateCancelQuery(page: playwright.Page) {
+  const queryOverview = new QueryOverview(page, UNIFIED_CONSOLE_URL);
+  const query = 'SELECT sleep(20)';
```

Comment:
@andreacyc is this 20 milliseconds or 20 seconds? If it's milliseconds, the test might be failing in CI because the query finishes before it's even possible to cancel it.

Reply:
it is 20 seconds.
i’m sorry that i rush this question into you. i was too greedy, wanted to access any possible help i can to fix this issue, and thought would be nice to have different angle from it. I know it now i can not do that, i will take some time slow down and do some experimental test for it.

Reply:
it's 20 seconds
i’m sorry that i rush this question into you. i was too greedy, wanted to access any possible help i can to fix this issue. I shouldn’t have you envolved this, i will take some time slow down and do some experimental test for it. And your comment do give me another thought about the response and waiting time, i will look into that!